automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -21,6 +21,9 @@
     satellite6_upgrade,
     satellite6_zstream_upgrade
 )
+from automation_tools.satellite6.upgrade.tasks import (
+    determine_satellite_version
```

Comment:
nitpick: would be good to rename `determine_satellite_version` to  `get_sat_version`